app/src/main/java/com/firstbuild/androidapp/dashboard/DashboardActivity.java

```diff
@@ -475,6 +481,42 @@ public void onClick(View v) {
             }
         });
 
+        findViewById(R.id.fiddleButton).setOnClickListener(new View.OnClickListener() {
```

Comment:
This is where the magic happens. This is throw away code, but it will be useful for when we actually start doing stuff.